packages/editor/src/components/rich-text/remove-browser-shortcuts.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * WordPress dependencies
+ */
+import { rawShortcut } from '@wordpress/keycodes';
+import { KeyboardShortcuts } from '@wordpress/components';
+
+export function RemoveBrowserShortcuts() {
```

Comment:
Is this removing from TinyMCE as well? How do _our_ handlers for Cmd+Z Undo not get removed as well?

The fact it works as-is all seems rather coincidental, not something I'd think to rely on.

Reply:
>Is this removing from TinyMCE as well?

Oh, I see these are removed in `TinyMCE`'s initialization.

Reply:
> > Is this removing from TinyMCE as well?
> 
> Oh, I see these are removed in `TinyMCE`'s initialization.

Clarified some in e69e6ff.